README.md

```diff
@@ -207,6 +207,46 @@ There is available REST API for AMOC where we can:
 * check status of nodes in AMOC cluster
 
 API is described [here](REST_API_DOCS.md). You can also get an access to REST API by running AMOC and go to `(address:port)/api-docs`.
-### Docker
 
-TODO
+## Docker
+
+To build a Docker image with Amoc, run the following command from the root of
+the repository:
+```
+docker build -f Dockerfile -t image .
+```
+It is important to start building at project root (it is indicated with dot `.`
+at the end of command). It will set build context at the project root. Dockerfile
+commands expects a context to be set like that:
+ - it copies **current** source code into container to compile it.
+ - It looks for files in `docker/` relative path.
+
+When image is ready it may be started just with
+```
+docker run -d image
+```
+
+However, you may want to use Amoc HTTP API for uploading and starting scenarios.
+In this case port 4000 should be published.
+```
+docker run -d -p 4000:4000 image
+```
+
+Amoc logs are connected with container standard output, so it is possible
+to get Amoc logs from running container with:
+```
+docker logs amoc_container
```

Comment:
This doesn't look consistent with other examples. Other examples didn't set container's name to `amoc_container`